.github/workflows/main.yml

```diff
@@ -3,18 +3,32 @@ on:
     types: [labeled]
 
 jobs:
-  trivial:
+  base:
     runs-on: ubuntu-latest
     if: contains(github.event.pull_request.labels.*.name, 'ci:run')
-    name: First Pass Trigger Testing
+    name: Main Tests
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
-      - name: Show Workspace Context
-        run: echo 'Workspace is ${{ github.workspace }}'
-      - name: Nasty Hack To Get Right Docker Build Context
-        run: cp ./.github/actions/docker_action/Dockerfile . && cp ./.github/actions/docker_action/action.yml .
-      - name: Run Docker Tests
-        uses: ./
-      - name: Show commit
-        run: git show -q
+      - name: test_all.sh running from Dockerfile.micro
+        uses: docker://jpwithers/tflite-micro-tests
```

Comment:
I'm assuming that we will be moving this to a docker image hosted with GitHub packages in a follow-up change?

I have created https://github.com/tensorflow/tflite-micro/issues/11 to start having open issues for work that we are punting for later. It would be nice to add a TODO(#11) comment in here as well.

Reply:
Yeah, I am still sorting through that now and getting it set up on my fork.